src/builddataset.py

```diff
@@ -0,0 +1,88 @@
+from sentiment import *
+import time
+import sys, os, csv
+
+class Dataset:
+    def __init__(self, topics):
+        self.topics = topics
+        self.profiles = dict() # Key: handle, Value: (label, {topic: (total score, count)}
+
+    # ingests the data from the input location
+    def ingest(self, filepath):
+        if not os.path.isfile(filepath):
+            print("File path {} does not exist. Exiting...".format(filepath))
+            sys.exit()
+
+
+        analyzer = Sentiment("apikey.json")
+        inputFile = open(filepath, "r")
+        while True:
+            info = inputFile.readline().strip("\n")
+
+            # if line is empty, end of file is reached
+            if not info:
+                break
+
+            tweeterInfo = info.split(",")
+            tweeterHandle, tweeterLabel = tweeterInfo[0], tweeterInfo[1]
+
+
+            numTweets = int(inputFile.readline().strip("\n"))
+            profileTopicsDict = dict()
+            for i in range(numTweets):
+                data = inputFile.readline().strip("\n").split(",")
+                text = data[0]
```

Comment:
So for some of the tweets, the value for text could be the string "empty" because they may just be a link to something so there's no text. I see you weigh the score by count, so maybe there should be a case where you skip over the text when text == empty

Reply:
The analyzer now gives a neutral score when there is no text.